### PR TITLE
Refractor CommentManagerDatastore tests to use TestConstants and override equals method in Comment for assertEquals

### PR DESCRIPTION
**Pull request by lisasgoh**

### CODE REVIEW

**Review by limli — DISMISSED** [review]

**Comment by limli** [comment]
this PR also fixes a test which failed after #68

**Review by jyx11011 — DISMISSED** [review]

File: src/test/java/com/google/step/TestConstants.java [hunk 1]
```diff
@@ -135,6 +135,9 @@
   public static final Comment COMMENT_B =
       new Comment(COMMENT_ID_B, DEAL_ID_A, USER_ID_B, CONTENT_B, TIME_B);
 
+  public static final Comment UPDATE_COMMENT_A =
```
Comment (SuyashLakhotia) [inline src/test/java/com/google/step/TestConstants.java]:
```suggestion
  public static final Comment UPDATED_COMMENT_A =
```

File: src/test/java/com/google/step/datamanager/CommentManagerDatastoreTest.java [hunk 2]
```diff
@@ -42,64 +41,50 @@ public void tearDown() {
...
-    assertEquals(USERID_A, commentAResult.userId);
-    assertEquals(CONTENT_A, commentAResult.content);
-    assertEquals(DEALID, commentBResult.dealId);
-    assertEquals(USERID_B, commentBResult.userId);
-    assertEquals(CONTENT_B, commentBResult.content);
+    Comment commentA = commentManagerDatastore.createComment(DEAL_ID_A, USER_ID_A, CONTENT_A);
+    Comment commentB = commentManagerDatastore.createComment(DEAL_ID_A, USER_ID_B, CONTENT_B);
+    List<Comment> comments = commentManagerDatastore.getCommentsForDeal(DEAL_ID_A);
+    assertEquals(commentA, comments.get(1));
+    assertEquals(commentB, comments.get(0));
```
Comment (SuyashLakhotia) [inline src/test/java/com/google/step/datamanager/CommentManagerDatastoreTest.java]:
```suggestion
    assert(comments, contains(commentA, commentB));
```

Does this work?

Comment (SuyashLakhotia) [reply src/test/java/com/google/step/datamanager/CommentManagerDatastoreTest.java]: Also assert on the size of the list?

File: src/test/java/com/google/step/datamanager/CommentManagerDatastoreTest.java [hunk 3]
```diff
@@ -42,64 +41,50 @@ public void tearDown() {
...
+    Comment commentA = commentManagerDatastore.createComment(DEAL_ID_A, USER_ID_A, CONTENT_A);
+    Comment commentB = commentManagerDatastore.createComment(DEAL_ID_A, USER_ID_B, CONTENT_B);
     commentManagerDatastore.deleteComment(commentA.id);
-    List<Comment> comments = commentManagerDatastore.getCommentsForDeal(DEALID);
-    Comment commentBResult = comments.get(0);
-    assertEquals(DEALID, commentBResult.dealId);
-    assertEquals(USERID_B, commentBResult.userId);
-    assertEquals(CONTENT_B, commentBResult.content);
+    List<Comment> comments = commentManagerDatastore.getCommentsForDeal(DEAL_ID_A);
+    assertEquals(commentB, comments.get(0));
```
Comment (SuyashLakhotia) [inline src/test/java/com/google/step/datamanager/CommentManagerDatastoreTest.java]: Similar here.

**Review by SuyashLakhotia — CHANGES_REQUESTED** [review]

File: src/test/java/com/google/step/datamanager/CommentManagerDatastoreTest.java (hunk 2 above)
Comment (lisasgoh) [reply src/test/java/com/google/step/datamanager/CommentManagerDatastoreTest.java]: I used hasItems

**Review by SuyashLakhotia — APPROVED** [review]

**Pull request merged**